frame/proxy/src/lib.rs

```diff
@@ -800,4 +796,10 @@ impl<T: Config> Pallet<T> {
 		let e = call.dispatch(origin);
 		Self::deposit_event(Event::ProxyExecuted { result: e.map(|_| ()).map_err(|e| e.error) });
 	}
+
+	pub fn remove_all_proxy_delegates(who: &T::AccountId) -> DispatchResult {
```

Comment:
This function doesn't contain any operation that can error, so it can just have no return value.

Reply:
As I understand, Do you mean to remove the DispatchResult  ?

Reply:
Yes.

Reply:
Done